src/platforms/android/androidvpnactivity.h

```diff
@@ -101,7 +101,7 @@ class AndroidVPNActivity : public QObject {
   void eventStatisticUpdate(const QString& data);
   void eventActivationError(const QString& data);
   void eventOnboardingCompleted();
-  void eventVpnConfigPermissionResponse();
+  void eventVpnConfigPermissionResponse(const QString& data);
```

Comment:
```
void eventVpnConfigPermissionResponse(bool ok);
```
Every time we use `const QString& data` i'd probably would assume it's a json string, just because "has always been". However doing that for "sucess", "fail" is ... excessive :D Let's just have this event be a bool.